storybook/stories/API/cartesian/YAxis.stories.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { ComponentProps, useState } from 'react';
+import { YAxis, XAxis, Legend, ResponsiveContainer, ComposedChart, Bar } from '../../../../src';
+import { pageData } from '../../data';
+
+export default {
+  component: YAxis,
+  argTypes: {
+    stroke: {
+      control: { type: 'color' },
+    },
+  },
+};
+
+const data = [
+  { name: 'Page A', pv: 800, amt: 1400 },
+  { name: 'Page B', pv: 967, amt: 1506 },
+  { name: 'Page C', pv: 1098, amt: 989 },
+  { name: 'Page D', pv: 1200, amt: 1228 },
+  { name: 'Page E', pv: 1108, amt: 1100 },
+  { name: 'Page F', pv: 680, amt: 1700 },
+];
+
+export const Simple = {
+  render: () => {
+    const allKeys = Object.keys(data[0]);
+    const [activeKeys, setActiveKeys] = useState(allKeys);
+
+    /*
+     * Toggles displayed bars when clicking on a legend item
+     */
+    const handleLegendClick: ComponentProps<typeof Legend>['onClick'] = e => {
+      const key: string = e.dataKey;
+      setActiveKeys(prev => (prev.includes(key) ? prev.filter(k => k !== key) : [...prev, key]));
+    };
+
+    return (
+      <ResponsiveContainer width={500} height={500}>
```

Comment:
I think since last review this changed in the storybook to use a fixed height and width: 100% - can you merge the latest and check what the pattern changed to?